### ISSUE TITLE
PHP Fatal error during environment set up

### ISSUE THREAD

**Issue by mckinley94c** [nunodonato/workspaice#5]
I am getting this error  when trying to set up the environment. I have no experience with PHP so this very well could be my own ignorance showing but it might be helpful if others come across the same issue.
php artisan key:generate
PHP Fatal error:  Uncaught Error: Class "Illuminate\Foundation\Application" not found in ~/workspaice/bootstrap/app.php:14
Stack trace:
#0 ~/workspaice/artisan(20): require_once()
#1 {main}
  thrown in ~/workspaice/bootstrap/app.php on line 14

Composer version 2.7.9 2024-09-04 14:43:28
PHP version 8.3.6 (/usr/bin/php8.3)

**Comment by nunodonato** [comment]
did you run composer install first?

**Comment by mckinley94c** [comment]
I went through some other errors I had been getting and it turns out I needed to install a handful of php extensions. I got up to the end and the http://127.0.0.1:8000/projects URL has a 500 server error. I am using firefox if that matters.

**Comment by mckinley94c** [comment]
I got it working hopefully this helps others if they come across similar issues even with other projects.
First, I tried doing a composer update which got me further along but not up and running yet.
Next, I made this change to the .env file DB_DATABASE=/absolute/path/to/database.sqlite
Then, I ran sudo apt install php8.3-sqlite3 (I am running php8.3 your mileage may vary for other versions) This had something to do the PDO not being enabled.
Finally I verified by using the command php -m | grep pdo_sqlite and it not being empty.
